CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ This document describes changes between each past release.
   intended to simplify the experience of people who are new to Kinto.
   Addresses #687.
 - Removed ``initialize_cliquet()``, which has been deprecated for a while.
+- Removed ``cliquet_protocol_version``. Kinto already defines API
+  variations as part of its URL format (e.g. ``/v0``,
+  ``/v1``). Services based on kinto.core are free to use
+  ``http_api_version`` to indicate any additional changes to their
+  APIs.
```

Comment:
I'm not sure what to think about that, tbh. I know we had already lenghty discussions about keeping the distinction between the protocol version defined and the actual version of kinto being used. IIRC, we wanted to keep them separate.

Keeping the protocol separated is a way to have some level of compatibility with kinto.core specific clients written in other languages, for instance. We wanted to have this to be future proof and in order to have a community around us, but I believe we failed at it so far.

As such, I wouldn't be too sad if we decide to remove it. What do @leplatrem and @natim think about this move?